src/Process/LayeredMaterialNodeProcessing.js

```diff
@@ -41,7 +41,7 @@ function initNodeImageryTexturesFromParent(node, parent, layer) {
 
 function initNodeElevationTextureFromParent(node, parent, layer) {
     // inherit parent's elevation texture
-    if (parent.material && parent.material.getElevationLayerLevel() > EMPTY_TEXTURE_ZOOM) {
+    if (parent.material && parent.material.getElevationLayerLevel() > node.material.getElevationLayerLevel()) {
```

Comment:
Is this change really needed?

(when would `initNodeElevationTextureFromParent` with node's elevation level not being EMPTY_TEXTURE_ZOOM)

Reply:
it's precisely this case that I avoid.
Because with two elevation layers we passe twice in `initNodeElevationTextureFromParent`

Reply:
Oh right. Then modify the comment above because it's not obvious.
Seomthing like:
```js
      // Inherit parent's elevation texture. Note that contrary to color layers the elevation level of the 
      // node might not be EMPTY_TEXTURE_ZOOM in this init function. That's because we can have
      // multiple elevation layers (thus multiple calls to initNodeElevationTextureFromParent) but a given
      // node can only use 1 elevation texture
```